apb.yml

```diff
@@ -1,5 +1,5 @@
 version: 1.0
-name: kubevirt-apb
+name: kubevirt
 description: APB for managing KubeVirt deployments
```

Comment:
How many times is the `Kubevirt` service expected to be provisioned?
Since it's cluster-wide, my understanding is `just once`.
To avoid confusion, I suggest to rephrase to sort of:
`APB for managing KubeVirt's deployment`